### ISSUE TITLE
Advanced in Admin settings

### ISSUE THREAD

**Issue by tim-rohrer** [overleaf/overleaf#1048]
## Steps to Reproduce

1. Create CE version of Overleaf.
2. Create first admin account and log in.
3. Click on Admin --> Manage Site.
4. Click on Advanced.

## Expected Behaviour

I would think we would be presented a list of advanced options to administer.

## Observed Behaviour
<img width="1187" alt="image" src="https://user-images.githubusercontent.com/7756865/193135642-8ecfca20-f940-4488-970c-f16479330d45.png">

## Context
Looking for more options to manage the site. In particular, I'd like see if we can turn off public registration.

## Technical Info

* URL: N/A
* Browser Name and version: Safari v16.0 and Chrome (latest)
* Operating System and version (desktop or mobile): macOS Desktop
* Signed in as:
* Project and/or file:

## Analysis
N/A

**Comment by mserranom** [comment]
> Looking for more options to manage the site. In particular, I'd like see if we can turn off public registration.

Would you mind being more specific on the list of options you'd like to see? Also, it would be better to have separate tickets per new feature, that enhances visibility.

**Comment by tim-rohrer** [comment]
At this time, I don't have any in mind other than the ability to turn off public registration. For that, I ended up using `nginx` to block the URL.

I view this as a bug since when I click on Advanced, nothing comes up on the screen, thus leaving the user to wonder what *should* be there.

If not having Advanced options is by design for the CE, I suggest 1) removing the Advanced tab, 2) adding the option I suggested (private vs public registration), or 3) adding a note that says advanced options aren't available for the CE, blah, blah, blah.

Thanks!
